scripts/bump-version.py

```
@@ -0,0 +1,71 @@
+    t = p.read_text(encoding="utf-8")
+    t, urls = re.subn(rf"\?v={re.escape(old)}\b", f"?v={new}", t)
+    t, stamp = re.subn(rf'(<a class="footer-version"[^>]*>)v{re.escape(old)}(</a>)',
+                       rf"\g<1>v{new}\g<2>", t)
+    t, label = re.subn(rf'(aria-label="Versión )v{re.escape(old)}', rf"\g<1>v{new}", t)
+    if not urls or not stamp:
+        sys.exit(f"{name}: expected asset URLs and a footer stamp at v{old}, "
+                 f"found {urls} and {stamp}")
+    p.write_text(t, encoding="utf-8")
+    print(f"{name}: {urls} asset URLs, footer stamp, aria-label ({label})")
+
+print(f"\nNow run: python3 scripts/check_pages.py")
```

Ruff (0.16.3)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)